primitives/api/proc-macro/Cargo.toml

```diff
@@ -21,8 +21,10 @@ syn = { version = "1.0.98", features = ["full", "fold", "extra-traits", "visit"]
 proc-macro2 = "1.0.37"
 blake2 = { version = "0.10.4", default-features = false }
 proc-macro-crate = "1.1.3"
+expander = "1.0.0"
+Inflector = "0.11.4"
```

Comment:
```suggestion
inflector = "0.11.4"
```

Reply:
https://crates.io/crates/Inflector I don't know if cargo is case insensitive :D

Reply:
Just checked, cargo is case sensitive.